magento/magento2
# Invoice sales email not send 13466

### Description (*)
send invoice email when orders are placed through paypal express.

### Fixed Issues (if relevant)
#13466: Invoice Sales Email not send
1. magento/magento2#<issue_number>: Issue title

### Manual testing scenarios (*)
Enable Paypal Express Checkout
Place Order using Paypal Express as Payment method
Check Invoice Email
### Contribution checklist (*)
